File: takasugi/Ruby/NLP-100-03.rb
#

org_str = "Now I need a drink, alcoholic of course, after the heavy lectures involving quantum mechanics."

puts org_str.split(/[\s,.]/).select{|s| s.size > 0}.map(&:size)

puts org_str.split(/[,. ]/).map(&:size).delete_if{|x| x == 0}

File: takasugi/Scala/NLP-100-03.scala
print("iii")
var str = "Now I need a drink, alcoholic of course, after the heavy lectures involving quantum mechanics."

println(str)

var o = str split("[ |,|.]")

var ob = o.toBuffer

var oob = ob.filterNot(e => e=="")
var ooob = oob.map(e => e.length)

//oob.foreach(e => println(e.length))



